docs/basic-guide-to-fhir-api-calls.md

```diff
@@ -0,0 +1,158 @@
+
+### Basic guide to FHIR Server REST calls:
+Once you have data in your FHIR server you can access it through REST API calls. Below are SOME examples of calls you can make.
+For additional information visit the [Microsoft FHIR API docs](https://learn.microsoft.com/en-us/azure/healthcare-apis/fhir/fhir-rest-api-capabilities) and/or the [HL7 FHIR Docs site](https://build.fhir.org/).
+
+#### Prerequisites:
+- Access to Azure FHIR server
+
+- Azure cli tool
+
+  - Auth Bearer token (autoexpires and may need regeneration every 60~90 mins)
+
+      - Generate Token by logging in with with `az login` then
+   
+        `az account get-access-token --resource={{FHIR-server-url}} --query accessToken --output tsv`
+
+- An API tool - Postman, Insomnia, or curl. Make sure to set up your auth header using the generated token.
+    - Postman Example Auth Header key value pair: `Authorization: Bearer <Your Token>`
+    - The end of Step 4 of this guide has a CURL example
+  
+- Shortcut for Postman Users:
+  - You can import the phdi-fhir-api-template.postman_collection.json in this directory into Postman to auto-load this collection.
+  - In Postman, import the collection with the Import button.
+  - Click the three dots by the collection name and select Edit
+  - Add your generated authorization token to the Authorization tab and your FHIR server url to the variables tab.
```

Comment:
Will also need to update all the id variables, reminder to add that here

Reply:
Not sure if you saw this since it wasn't part of a review, flagging now @gordonfarrell